code/modules/mob/living/brain/brain_item.dm

```diff
@@ -137,7 +138,7 @@
 		. += "<span class='info'>It's started turning slightly grey. They must not have been able to handle the stress of it all.</span>"
 	else if(brainmob)
 		if(brainmob.get_ghost(FALSE, TRUE))
-			if(brain_death || brainmob.health <= HEALTH_THRESHOLD_DEAD)
+			if(brainmob.health <= HEALTH_THRESHOLD_DEAD)
 				. += "<span class='info'>It's lifeless and severely damaged.</span>"
```

Comment:
Would be nice if there was a step between "no message" and "brain is 100% dead", like "it's visibly damaged" at half health or something

Reply:
This was a mess really, cleaned up the entire thing.